### PR TITLE
Main prettified

### PR DESCRIPTION
**Pull request by kvz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,7 +44,9 @@
     "fakefile": "^0.0.9",
     "jest": "^26.6.3",
     "nock": "^13.0.5",
+    "npm-run-all": "^4.1.5",
```
Comment (mifi) [inline package.json:47]: what's this for?

File: src/Transloadit.js [hunk 2]
```diff
@@ -143,24 +148,27 @@ class TransloaditClient {
...
+            // https://github.com/transloadit/node-sdk/issues/92
+            throw new Error(`Upload named "${label}" is not a Readable stream`)
+          }
 
-        return [
-          label,
-          isStream.readable(value) ? value : intoStream(value),
-        ]
-      }))
+          return [label, isStream.readable(value) ? value : intoStream(value)]
```
Comment (mifi) [inline src/Transloadit.js:164]: not sure if this is more readable than the previous 😅

File: types/index.test-d.ts [hunk 3]
```diff
@@ -23,59 +30,69 @@ const transloadit = new Transloadit({
...
+)
+expectType<Readable>(transloadit.streamAssemblies({ param1: { a: 1 } }))
 expectType<Promise<Assembly>>(transloadit.getAssembly('1'))
 
-expectType<Promise<TemplateResponse>>(transloadit.createTemplate({ param1: { a: 1 }}))
-expectType<Promise<TemplateResponse>>(transloadit.editTemplate('1', { param1: { a: 1 }}))
-expectType<Promise<{ ok: string, message: string }>>(transloadit.deleteTemplate('1'))
+expectType<Promise<TemplateResponse>>(transloadit.createTemplate({ param1: { a: 1 } }))
+expectType<Promise<TemplateResponse>>(transloadit.editTemplate('1', { param1: { a: 1 } }))
+expectType<Promise<{ ok: string; message: string }>>(transloadit.deleteTemplate('1'))
```
Comment (mifi) [inline types/index.test-d.ts:89]: are comma and semicolon interchangeable in typescript?

File: .prettierrc.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "http://json.schemastore.org/prettierrc",
+  "printWidth": 100,
```
Comment (mifi) [inline .prettierrc.json:3]: prettier recommends against setting this:
https://prettier.io/docs/en/options.html

is there a good reason for setting it higher?

**Review by mifi — COMMENTED** [review]
Some of it takes some getting used to but other than that should be good! I see that prettier disables a lot of our transloadit rules. hopefully none of them are important (I didn't go thru and check)

File: README.md [hunk 5]
```diff
@@ -137,26 +144,27 @@ You can provide the following keys inside the `options` object:
...
   - `uploadedBytes` - Number of bytes uploaded so far.
   - `totalBytes` - Total number of bytes to upload or `undefined` if unknown (Streams).
-- `onAssemblyProgress` - Once the Assembly has started processing this will be periodically called with the *Assembly Execution Status* (result of `getAssembly`) **only if `waitForCompletion` is `true`**.
+- `onAssemblyProgress` - Once the Assembly has started processing this will be periodically called with the _Assembly Execution Status_ (result of `getAssembly`) **only if `waitForCompletion` is `true`**.
 - `chunkSize` - (for uploads) a number indicating the maximum size of a tus `PATCH` request body in bytes. Default to `Infinity` for file uploads and 50MB for streams of unknown length. See [tus-js-client](https://github.com/tus/tus-js-client/blob/master/docs/api.md#chunksize).
 - `uploadConcurrency` - Maximum number of concurrent tus file uploads to occur at any given time (default 10.)
 
- **NOTE**: Make sure the key in `files` and `uploads` is not one of `signature`, `params` or `max_size`.
+  **NOTE**: Make sure the key in `files` and `uploads` is not one of `signature`, `params` or `max_size`.
```
[the remark this answers is not in the document]
Comment (kvz) [reply README.md:164]: could be a bug 🤔

File: package.json (hunk 1 above)
Comment (kvz) [reply package.json:47]: Ah, it's to make linting run in parallel to save time, but i didn't wire it up yet. Did that just now 👌

File: src/Transloadit.js (hunk 2 above)
Comment (kvz) [reply src/Transloadit.js:164]: Sadly, in some cases the autofixing results in worse code. I had this in the content repo too. It's a downside. All I can say is that after biting the bullet

A) the upsides weigh up to the few cases
B) it's easy to clean these cases up a bit again as you work on the code and get to know what prettier will be doing to it

File: types/index.test-d.ts (hunk 3 above)
Comment (kvz) [reply types/index.test-d.ts:89]: no expert, in this case they are :)

File: .prettierrc.json (hunk 4 above)
Comment (mifi) [reply .prettierrc.json:3]: you mean higher printWidth causes lines to break up less, right? not sure why a higher printWidth would reduce the number of long, unbroken lines

File: src/Transloadit.js (hunk 2 above)
Comment (kvz) [reply src/Transloadit.js:164]: yes, entire files, or: ignore the next expression.

File: .prettierrc.json (hunk 4 above)
Comment (kvz) [reply .prettierrc.json:3]: yes but then there will be extra cases that raises eyebrows, or so was our experience in content which led to 100 as a seemingly reasonable alternative

**Review by mifi — APPROVED** [review]

**Pull request merged**